packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -432,13 +498,21 @@ mixin SchedulerBinding on BindingBase {
     if (_hasRequestedAnEventLoopCallback)
       return;
     _hasRequestedAnEventLoopCallback = true;
-    Timer.run(_runTasks);
+    scheduleTasks(_runTasks);
+  }
+
+  /// Schedules tasks outside of the current event loop.
+  ///
+  /// By default this uses [Timer.run].
+  @protected
+  void scheduleTasks(void Function() callback) {
```

Comment:
The method name and dartdocs refer to "tasks" but it's unclear what those tasks are.